src/styles/blocks/clock.scss

```
@@ -0,0 +1,60 @@
+.clock {
+  @extend %centered-content;
+
+  position: relative;
+
+  width: $clockWidth;
+  height: $clockHeight;
+  border: 4px solid darkred;
+  border-radius: 50%;
+  background-color: beige;
```

Would be better if you create variables for colors
```suggestion
  background-color: $c-beige;
```